runtime/vam/expr/function/has.go

```diff
@@ -0,0 +1,100 @@
+package function
+
+import (
+	"github.com/RoaringBitmap/roaring"
+	"github.com/brimdata/super"
+	"github.com/brimdata/super/runtime/vam/expr"
+	"github.com/brimdata/super/vector"
+)
+
+type Has struct {
+	missing Missing
+	not     *expr.Not
+}
+
+func newHas(zctx *super.Context) *Has {
+	return &Has{not: expr.NewLogicalNot(zctx, &expr.This{})}
+}
+
+func (h *Has) Call(args ...vector.Any) vector.Any {
+	return h.not.Eval(h.missing.Call(args...))
+}
+
+type Missing struct{}
+
+func (m *Missing) Call(args ...vector.Any) vector.Any {
+	n := args[0].Len()
+	var nbm roaring.Bitmap
+	for _, vec := range args {
+		if nulls := vector.NullsOf(vec); nulls != nil {
+			nbm.Or(roaring.FromDense(nulls.Bits, false))
+		}
+		if err, ok := vec.(*vector.Error); ok {
+			b := missingOrQuietV2(err)
+			if b.IsEmpty() {
+				return err
+			}
+			if b.GetCardinality() == uint64(n) {
+				return vector.NewConst(super.True, vec.Len(), bitmapToBool(&nbm, n))
+			}
+			// Mix of errors and trues.
+			index := b.ToArray()
+			errIndex := roaring.Flip(b, 0, uint64(n)).ToArray()
+			trueVec := vector.NewConst(super.True, uint32(len(index)), nil)
+			if !nbm.IsEmpty() {
+				trueVec.Nulls = vector.NewView(bitmapToBool(&nbm, n), index).(*vector.Bool)
+			}
+			return vector.Combine(trueVec, errIndex, vector.NewView(err, errIndex))
+		}
+	}
+	return vector.NewConst(super.False, args[0].Len(), bitmapToBool(&nbm, n))
+}
+
+func bitmapToBool(b *roaring.Bitmap, len uint32) *vector.Bool {
+	if b.IsEmpty() {
+		return nil
+	}
+	return vector.NewBool(b.ToDense(), len, nil)
+}
+
+func missingOrQuietV2(verr *vector.Error) *roaring.Bitmap {
```

Comment:
Nit: Why V2? How about isMissingOrQuiet?

Reply:
ha my bad at some point this was v2